src/main/java/net/minecraftforge/common/ForgeHooks.java

```diff
@@ -390,7 +390,7 @@ public static int getLootingLevel(Entity target, @Nullable Entity killer, Damage
         int looting = 0;
         if (killer instanceof LivingEntity)
             looting = EnchantmentHelper.getMobLooting((LivingEntity)killer);
-        if (target instanceof LivingEntity)
+        if (target instanceof LivingEntity && killer != null)
```

Comment:
The killer can be explicitly null. Firing the looting level event is explicitly intended to allow handeling when the killer is null so things can have a base looting level, or damage sources can have a looting level.

Reply:
Thank you for the review! So you think the predicate should be removed again? My reason for adding it was that vanilla requires the KILLER_ENTITY loot parameter for all looting related things, so it seemed reasonable to assume that looting is only relevant if there is a killer. But of course that is debatable, as you pointed out.